src/index.html

```diff
@@ -13,6 +13,78 @@
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <container class="container">
+      <div class="stars stars--0">
+        <div class="stars__star"></div>
+
```

Comment:
We don't need empty lines between `stars__star` elements. According to the guideline, they should be added between multiline siblings - and `stars__star` are single-line.